samples/open-telemetry/customizing/Core_9/Sample/Program.cs

```diff
@@ -3,64 +3,55 @@
 using OpenTelemetry.Resources;
 using OpenTelemetry.Trace;
 using System;
-using System.Threading.Tasks;
 
-public static class Program
-{
-    public static async Task Main()
-    {
-        Console.Title = "CustomTelemetry";
+Console.Title = "MyEndpoint";
+
+#region open-telemetry-config
+
+var resourceBuilder = ResourceBuilder.CreateDefault()
+    .AddService("MyEndpoint");
+
+using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+    .SetResourceBuilder(resourceBuilder)
+    .AddSource("NServiceBus.Core")
+    .AddSource(CustomActivitySources.Name)
+    .AddProcessor(new NetHostProcessor())
+    .AddConsoleExporter()
+    .Build();
 
-        #region open-telemetry-config
-        var resourceBuilder = ResourceBuilder.CreateDefault()
-            .AddService("CustomTelemetry");
+#endregion
 
-        var tracerProviderBuilder = Sdk.CreateTracerProviderBuilder()
-            .SetResourceBuilder(resourceBuilder)
-            .AddSource("NServiceBus.Core*")
-            .AddSource(CustomActivitySources.Name)
-            .AddProcessor(new NetHostProcessor())
-            .AddConsoleExporter();
-        #endregion
+#region enable-opentelemetry
 
-        using (var tracerProvider = tracerProviderBuilder.Build())
-        {
-            #region enable-opentelemetry
-            var endpointConfiguration = new EndpointConfiguration("CustomTelemetry");
-            endpointConfiguration.EnableOpenTelemetry();
-            #endregion
-            endpointConfiguration.UseSerialization<SystemJsonSerializer>();
-            endpointConfiguration.UseTransport<LearningTransport>();
+var endpointConfiguration = new EndpointConfiguration("MyEndpoint");
+endpointConfiguration.EnableOpenTelemetry();
 
-            endpointConfiguration.Pipeline.Register(
-                new TraceOutgoingMessageSizeBehavior(),
-                "Captures body size of outgoing messages as OpenTelemetry tags"
-            );
+#endregion
 
-            var endpoint = await Endpoint.Start(endpointConfiguration);
```

Comment:
I thought about using extensions hosting but I don't think it would have made the code more terse

```
// Initialize OpenTelemetry
builder.Services.AddOpenTelemetry()
    .ConfigureResource(resource => /* Resource configuration goes here */)
    .WithLogging(logging => /* Logging configuration goes here */)
    .WithMetrics(metrics => /* Metrics configuration goes here */)
    .WithTracing(tracing => /* Tracing configuration goes here */));
```